parsers/mongodb/mongodb_test.go

```diff
@@ -19,6 +19,13 @@ const (
+	MONGO_3_4_GETMORE   = `2016-10-20T22:28:01.785+0000 I COMMAND  [conn8] command TestDB.TestColl appName: "MongoDB Shell" command: getMore { getMore: 18862806827, collection: "TestColl" } originatingCommand: { find: "TestColl", projection: { _id: 0.0, Counter: 1.0 }, shardVersion: [ Timestamp 1000|0, ObjectId('580944efeaec999c2d8e0d5b') ] } planSummary: COLLSCAN cursorid:18862806827 keysExamined:0 docsExamined:59899 cursorExhausted:1 numYields:468 nreturned:59899 reslen:1726100 locks:{ Global: { acquireCount: { r: 938 } }, Database: { acquireCount: { r: 469 } }, Collection: { acquireCount: { r: 469 } } } protocol:op_command 120ms`
+	MONGO_3_4_INIT      = `2016-10-20T21:13:03.294+0000 I CONTROL [initandlisten] options: { net: { port: 23511 }, nopreallocj: true, replication: { oplogSizeMB: 40, replSet: "test-configRS" }, setParameter: { enableTestCommands: "1", logComponentVerbosity: "{tracking:1}", numInitialSyncAttempts: "1", numInitialSyncConnectAttempts: "60", writePeriodicNoops: "false" }, sharding: { clusterRole: "configsvr" }, storage: { dbPath: "/data/db/job14/mongorunner/test-configRS-0", engine: "wiredTiger", journal: { enabled: true }, mmapv1: { preallocDataFiles: false, smallFiles: true }, wiredTiger: { engineConfig: { cacheSizeGB: 1.0 } } }}`
+	MONGO_3_4_INDEX     = `2016-10-20T22:27:59.508+0000 I INDEX    [conn5] build index on: TestDB.TestColl properties: { v: 2, key: { Counter: 1.0 }, name: "Counter_1", ns: "TestDB.TestColl" }`
+	MONGO_3_4_SHARDING  = `2016-10-20T22:27:59.516+0000 I SHARDING [conn1] about to log metadata event into changelog: { _id: "ip-10-69-189-27-2016-10-20T22:27:59.516+0000-580944efeaec999c2d8e0d58", server: "ip-10-69-189-27", clientAddr: "127.0.0.1:35756", time: new Date(1477002479516), what: "shardCollection.start", ns: "TestDB.TestColl", details: { shardKey: { Counter: 1.0 }, collection: "TestDB.TestColl", primary: "shard0000:ip-10-69-189-27:23760", initShards: [], numChunks: 1 } }`
+	UPDATE_COMMAND      = `Tue Sep 13 21:10:33.961 I COMMAND  [conn11896572] command data.$cmd command: update { update: "avengers", updates: [ { q: { hulkForm: "Bruce Banner" }, u: { $set: { hulkForm: "Big Green" }, $setOnInsert: { hulkForm: "Big Green" } }, upsert: true } ], writeConcern: { getLastError: 1, w: 1 }, ordered: true } keyUpdates:0 writeConflicts:0 numYields:0 reslen:95 locks:{ Global: { acquireCount: { r: 1, w: 1 } }, Database: { acquireCount: { w: 1 } }, Collection: { acquireCount: { w: 1 } } } user_key_comparison_count:466 block_cache_hit_count:10 block_read_count:0 block_read_byte:0 internal_key_skipped_count:17 internal_delete_skipped_count:0 get_from_memtable_count:0 seek_on_memtable_count:2 seek_child_seek_count:12 0ms`
```

Comment:
in the interest of using the test cases as a "here's what to expect for X type of log line," think we can get simpler update / delete cases represented alongside these multi-body ones?